charts/nginx/templates/nginx-configmap.yaml

```diff
@@ -48,14 +48,16 @@ data:
 
         -- Split the query string into individual parameters
         for param in string.gmatch(queryString, "([^&]+)") do
-          local paramName, paramValue = string.match(param, "(.+)=(.+)")
+          local paramName, paramValue = string.match(param, "(.+)=(.*)") -- Allow empty paramValue
 
           -- Check if the parameter should be masked
           if paramsToMask[paramName] then
-              paramValue = "REDACTED" -- Replace the value with REDACTED
+              paramValue = "##REDACTED##" -- Replace the value with REDACTED
           end
 
-          table.insert(queryParams, paramName .. "=" .. paramValue)
+          if paramName and paramName ~= '' and paramValue then -- Check if paramName and paramValue exist
```

Comment:
wouldn't `paramName ~= ''` only match a blank `paramName`?

Reply:
if paramName and paramValue then -- this should be enough we dont need to check paramName to be empty is that okay @danielhoherd

Reply:
it seems like that would work. the regex specifies that the paramName must be at least one character.